vtex-sites/nextjs.store
# feat: Extract `RegionalizationModal`

Signed-off-by: Arthur Andrade <arthurfelandrade@gmail.com>## What's the purpose of this pull request?

## What's the purpose of this pull request?

- Extracts `Modal` component from `RegionalizationModal`
- Applies the new https://github.com/vtex-sites/base.store/pull/407 to `Modal` and `RegionalizationModal`: new scoped tokens linked to global ones.

## How does it work?

|Before|Now|
|-|-|
![Screen Shot 2022-06-16 at 14 02 01](https://user-images.githubusercontent.com/51174217/174126299-6145602a-bf46-48e8-a544-0551b4f1cd20.png)|![Screen Shot 2022-06-16 at 13 58 45](https://user-images.githubusercontent.com/51174217/174125790-9e4659c9-56f3-473e-834a-f08ee20ae49e.png)

Tokens applied for `RegionalizationModal` and `Modal`.

## How to test it?

- Check the `RegionalizationModal` documentation in the [preview](https://nextjs-store-storybook-git-feat-add-tokens-to-b3ff85-faststore.vercel.app/?path=/docs/features-regionalization-regionalizationmodal--overview) 
- Check the `Modal`  documentation in the [preview](https://nextjs-store-storybook-git-feat-add-tokens-to-b3ff85-faststore.vercel.app/?path=/docs/molecules-modal--overview)

## Checklist

<em>You may erase this after checking them all ;)</em>

**Changelog**

**PR Description**

**Documentation**
